deploy/chart/templates/daemonset.yaml

```diff
@@ -35,7 +37,11 @@ spec:
       tolerations:
       {{- toYaml . | nindent 8 }}
       {{- end }}
-      hostNetwork: {{ default true .Values.enableHostNetwork }}
+      {{- if ne .Values.enableHostNetwork false }}
+      hostNetwork: true
+      {{- else }}
+      hostNetwork: false
+      {{- end }}
```

Comment:
Do we need this change? Can we just make sure we have false and not "false" in values.yaml to make sure its set correctly?

Reply:
fair enough. I was trying to get the value commented in values.yaml. But I think having a definition is much more simpler.